Gems/AtomLyIntegration/CommonFeatures/Code/Source/Material/MaterialBrowserInteractions.cpp

```diff
@@ -39,6 +46,61 @@ namespace AZ
+        {
+            using namespace AtomToolsFramework;
+
+            creators.push_back(
+                { "Material_Creator", "Material", QIcon(),
+                  [&]([[maybe_unused]] const char* fullSourceFolderNameInCallback, [[maybe_unused]] const AZ::Uuid& sourceUUID)
+                  {
+                      const AZ::Data::AssetId assetId = AtomToolsFramework::GetSettingsObject<AZ::Data::AssetId>(
+                          "/O3DE/Atom/MaterialEditor/DefaultMaterialTypeAsset",
+                          AZ::RPI::AssetUtils::GetAssetIdForProductPath("materials/types/standardpbr.azmaterialtype"));
+                      QWidget* mainWindow = nullptr;
+                      AzToolsFramework::EditorRequests::Bus::BroadcastResult(
+                          mainWindow, &AzToolsFramework::EditorRequests::Bus::Events::GetMainWindow);
+
+                      CreateDocumentDialog dialog(
+                          QObject::tr("Create Material"), QObject::tr("Select Type"), QObject::tr("Select Material Path"),
+                          fullSourceFolderNameInCallback, { "material" }, assetId,
+                          []([[maybe_unused]] const AZ::Data::AssetInfo& assetInfo)
+                          {
+                              AZStd::unordered_set<AZ::Uuid> supportedAssetTypesToCreate;
+                              supportedAssetTypesToCreate.insert(azrtti_typeid<AZ::RPI::MaterialTypeAsset>());
+
+                              // If any asset types are specified, do early rejection tests to avoid expensive string comparisons
+                              const auto& assetTypes = supportedAssetTypesToCreate;
+                              if (assetTypes.empty() || assetTypes.find(assetInfo.m_assetType) != assetTypes.end())
+                              {
+                                  // Additional filtering will be done against the path to the source file for this asset
+                                  const auto& sourcePath = AZ::RPI::AssetUtils::GetSourcePathByAssetId(assetInfo.m_assetId);
+
+                                  // Only add source files with extensions supported by the document types creation rules
+                                  // Ignore any files that are marked as non editable in the registry
+                                  return AZ::StringFunc::EndsWith(sourcePath.c_str(), AZ::RPI::MaterialTypeSourceData::Extension) &&
+                                      IsDocumentPathEditable(sourcePath);
+                              }
+                              return false;
+                          }, mainWindow);
+                      dialog.adjustSize();
+
+                      if (dialog.exec() == QDialog::Accepted && !dialog.m_sourcePath.isEmpty() && !dialog.m_targetPath.isEmpty())
+                      {
+                          AZ::RPI::MaterialSourceData materialData;
+                          materialData.m_materialTypeVersion = 5;
```

Comment:
This version number needs to come from the material type. You might have to quickly read in the material type source data structure to get the version number. Once this is submitted we'll go back and create a utility function for it.

Reply:
Changed as requested.